interfaces/builtin/network_status.go

```diff
@@ -24,131 +24,82 @@ import (
 
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/interfaces/apparmor"
-	"github.com/snapcore/snapd/interfaces/dbus"
 	"github.com/snapcore/snapd/snap"
 )
 
-const networkStatusSummary = `allows operating as the NetworkingStatus service`
+const networkStatusSummary = `allows access to network connectivity status`
 
 const networkStatusBaseDeclarationSlots = `
   network-status:
     allow-installation:
       slot-snap-type:
         - app
```

Comment:
If this is only implicit, we should remove `- app`. There is only one snap in the store that slots this, and it is the snap created for our spread tests.

Reply:
Fair enough.  I suppose the stubs won't really be that helpful should we try to extend this to handle Ubuntu Core.

Reply:
Do you have any thoughts on whether this interface should auto connect in its new form?

Reply:
I do believe it should auto-connect, yes. I forgot to mention that since '- app' is removed, we don't want the deny-connection. The full base declaration should be:
```
  network-status:
    allow-installation:
      slot-snap-type:
        - core
```